src/web/app/pages-student/student-profile-page/student-profile-page.component.ts

```diff
@@ -124,6 +124,10 @@ export class StudentProfilePageComponent implements OnInit {
     const modalRef: NgbModalRef = this.ngbModal.open(UploadEditProfilePictureModalComponent);
     modalRef.componentInstance.profilePicLink = this.profilePicLink;
     modalRef.result.then((formData: FormData) => {
+      if (!formData) {
+        this.statusMessageService.showWarningMessage('No photo detected. You maybe uploaded an empty photo');
```

Comment:
Could we improve on the error message here? What does it mean to say 'you maybe uploaded an empty photo'?

Why not something like 'No photo uploaded.'?